tests/conversion/downcast-converters.js

```diff
@@ -686,6 +686,26 @@ describe( 'downcast-converters', () => {
 			// No attribute set.
 			expect( viewToString( viewRoot ) ).to.equal( '<div><p>foobar</p></div>' );
 		} );
+
+		it( 'should not convert or consume if element creator returned null', () => {
+			const spy = sinon.spy();
```

Comment:
I'd use `const callback = sinon.stub().returns( null )` here - tests will be shorter.